cookbook/persistance/key-value.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: "Storing key-value data on disk"
+permalink: /cookbook/persistence/key-value/
+---
+
+If you have a relatively small collection of key-values that you'd like to save, you can use the [shared_preferences](https://pub.dartlang.org/packages/shared_preferences) plugin.
```

Comment:
The text in this recipe switches between "you" and "we". Could you please use "we" throughout? I'm not sure it's the best choice, but I've gone with "we" in other recipes, and good to be consistent.